akka-cluster-sharding/src/main/scala/akka/cluster/sharding/ShardCoordinator.scala

```diff
@@ -951,6 +951,7 @@ abstract class ShardCoordinator(
   def handleGetShardHome(shard: ShardId): Boolean = {
```

Comment:
@octonato I have moved the unstashing to handleGetShardHome. Take a look if that matches what you were thinking of. commit for that change is 1af2c1e366

Reply:
Looks better now, thanks.